src/main/java/org/broadinstitute/hellbender/utils/gcs/BucketUtils.java

```diff
@@ -374,6 +374,17 @@ public static String getPathWithoutBucket(String path) {
 
     }
 
+    /**
+     * String -> Path. This *should* not be necessary (use Paths.get(URI.create(...)) instead) , but it currently is
+     * on Spark because using the fat, shaded jar breaks the registration of the GCS FilesystemProvider.
```

Comment:
Upon investigation it looks like META-INF/services is fine (and the jar
shader's doing the right thing). The problem's elsewhere.